### PR TITLE
getting involved section more newcomer-friendly

### PR DESCRIPTION
**Pull request by MattAllmendinger**
involve section is more newcomer-friendly now

### CODE REVIEW

**Comment by tushar-rishav** [comment]
@MattAllmendinger Imperative present tense for commit.
More info at http://coala.readthedocs.org/en/latest/Getting_Involved/Writing_Good_Commits.html#shortlog

**Comment by tushar-rishav** [comment]
Rest is clear from Gitmate logs. Hope to see get fixed too

**Comment by tushar-rishav** [comment]
unack dfdac94

**Comment by tushar-rishav** [comment]
@MattAllmendinger Running `coala` on your code would fix most of the errors here. :D

**Comment by tushar-rishav** [comment]
@MattAllmendinger You might not need to create a new commit for each fix. You could just `git commit --amend`

**Comment by tushar-rishav** [comment]
unack

**Comment by SanketDG** [comment]
cbe7a21 needs work

**Comment by MattAllmendinger** [comment]
How can I change the shortlog of the HEAD commit?

**Comment by MattAllmendinger** [comment]
unack d37083d

File: README.rst [hunk 1]
```diff
@@ -79,16 +79,20 @@ Project Status
...
-Getting Involved
-----------------
+Newcomers Guide and Getting Involved
+------------------------------------
 
-If you want to contribute to coala, please take a look at the `Getting
-Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+coala is newcomer friendly and we have built an entire website just
+to get any contribution done really easily.
```
Comment (sils1297) [inline README.rst]: Hi,

I took the freedom and deleted the cluttering GitMate messages.

I find this "and we have build an entire website just to get any contribution done really easily" not quite satisfying:

1. It's just a documentation page, not an "entire website".
2. It sounds a bit like "we've done something great! Now be grateful! And thank us! Send us chocolate!"
3. "really easily" doesn't sound quite right to me.

(I know that this was phrased by @Adrianzatreanu here but that doesn't change that IMO it shouldn't be accepted. FWIW some contributions take longer, most others had better defined tasks, some had like a typo or something so even across newcomer issues there's some differences.)

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/1985/commits/8dc3cb5a21ebc2ad69cb05af5971b0b566ed0011 needs work

**Comment by sils1297** [comment]
Commit message: `Whitespace removed` - there was no whitespace before your commit to remove ;)

**Comment by sils1297** [comment]
@MattAllmendinger you committed a `.DS_Store` file apparently, that shouldn't be there.

File: README.rst [hunk 2]
```diff
@@ -79,16 +79,19 @@ Project Status
...
 
-Getting Involved
-----------------
+Newcomers Guide and Getting Involved
+------------------------------------
 
-If you want to contribute to coala, please take a look at the `Getting
-Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+coala is newcomer friendly and you will find a documentation
```
Comment (sils1297) [inline README.rst]: a documentation? Sounds cute, you usually say `you will find documentation` or `you fill find a documentation page`. No really IMO this whole paragraph shouldn't be here, @Adrianzatreanu what was the reason for you to want to include it? It holds no additional info, everything important is covered below.

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala/pull/1985/commits/8be6c55ff45cf9f121621f4daada1ba8c13ade8a

**Comment by sils1297** [comment]
please squash the commits ;)

**Comment by MattAllmendinger** [comment]
@sils1297 hope I squashed them in the right order ;)

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (sils1297) [inline .gitignore]: this change should go into an own commit, also you still add the actual file ;)

**Comment by sils1297** [comment]
your commit message is still a bit messy but we're getting to it I think :)

**Comment by sils1297** [comment]
can you come by at :point_up: [March 31, 2016 6:51 PM](https://gitter.im/coala-analyzer/coala/workshops?at=56fd55a11a1ccc164832a4c8) ?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/1985/commits/09a91c18e97fec6fd3e61c1b5736e5203dfa2b25

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/1985/commits/d3e936fc7f546745fb4c2be4b69c955fc0950dab, will merge it with the batch of the other commits

**Comment by sils1297** [comment]
included in https://github.com/coala-analyzer/coala/pull/1989
